raster-foundry/raster-foundry
# Add support for scalafmt via set plugin

## Overview

Adds `scalafmt` support via `sbt-scalafmt` to `test` script, which will fail when source code is not formatted correctly. In addition, formats all `.scala` and `.sbt` source files with `scalafmt.

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] [Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~


~- [ ] Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
~- [ ] Any new SQL strings have tests~

### Notes

See: https://scalameta.org/scalafmt/

## Testing Instructions

- Reformat the source code in a file
- Execute `test` to see `scalafmt` in action
